src/app/tests/suites/certification/Test_TC_OPCREDS_3_7.yaml

```diff
@@ -124,8 +108,8 @@ tests:
 
       # verification: ""
     - label:
-          "Step 6.2: TH2 generates the NOC, the Root CA Certificate and ICAC
-          using the CSR elements from Step 5 and selects an IPK, all for use by
+          "Step 4.1: TH2 generates the NOC, the Root CA Certificate and ICAC
+          using the CSR elements from Step 3 and selects an IPK, all for use by
           TH2. Save ICAC as Intermediate_Certificate_TH2. Save NOC as
```

Comment:
```suggestion
          Save ICAC as Intermediate_Certificate_TH2. Save NOC as
```

Reply:
Fixed: The step description has been updated, and 4.1 has been changed to 4 due to a related sequence adjustment.